VERSION

```diff
@@ -1 +1 @@
-0.2.0
```

Comment:
@SamuelSarle PRs should never mix version bumps into them, they should only implement some actual change